src/main/java/org/mitre/inferno/rest/ValidatorEndpoint.java

```diff
@@ -39,11 +39,17 @@ public static ValidatorEndpoint getInstance(Validator validator) {
    * Creates the API routes for receiving and processing requests for the validation service.
    */
   private void createRoutes() {
-    post("/validate",
```

Comment:
Use cases:

```
/validate
/Patient/$validate --> /WILDCARD/$validate
/Patient/123/$validate --> /WILDCARD/WILDCARD/$validate
```
Where WILDCARD represents possible resourceTypes (2nd case), as well as an ID (3rd case). The question is whether the WILDCARD representing ID in the 3rd case will be consumed by a singular WILDCARD.